### PR TITLE
channel `name` mechanism

### PR DESCRIPTION
**Pull request by BreadcrumbIsTaken**
## `channel_name` mechanism

For `DiscordChannelTag`s. Changes the name of a channel.

Requested by Hedwy on [Discord](https://discord.com/channels/315163488085475337/1082790105162461294)

### CODE REVIEW

File: src/main/java/com/denizenscript/ddiscordbot/objects/DiscordChannelTag.java [hunk 1]
```diff
@@ -575,5 +575,16 @@ public void adjust(Mechanism mechanism) {
         if (mechanism.matches("delete")) {
             getChannel().delete().complete();
         }
+
+        // <--[mechanism]
+        // @object DiscordChannelTag
+        // @name channel_name
+        // @input ElementTag
+        // @description
+        // Renames this channel
```
Comment (mcmonkey4eva) [inline src/main/java/com/denizenscript/ddiscordbot/objects/DiscordChannelTag.java]: period at end of sentence

File: src/main/java/com/denizenscript/ddiscordbot/objects/DiscordChannelTag.java [hunk 2]
```diff
@@ -575,5 +575,16 @@ public void adjust(Mechanism mechanism) {
...
         }
+
+        // <--[mechanism]
+        // @object DiscordChannelTag
+        // @name channel_name
+        // @input ElementTag
+        // @description
+        // Renames this channel
+        // -->
+        if (mechanism.matches("channel_name")) {
```
Comment (mcmonkey4eva) [inline src/main/java/com/denizenscript/ddiscordbot/objects/DiscordChannelTag.java]: missing input requirement check, or any other input validation

File: src/main/java/com/denizenscript/ddiscordbot/objects/DiscordChannelTag.java [hunk 3]
```diff
@@ -575,5 +575,16 @@ public void adjust(Mechanism mechanism) {
         if (mechanism.matches("delete")) {
             getChannel().delete().complete();
         }
+
+        // <--[mechanism]
+        // @object DiscordChannelTag
+        // @name channel_name
```
Comment (mcmonkey4eva) [inline src/main/java/com/denizenscript/ddiscordbot/objects/DiscordChannelTag.java]: `channel.channel_name` is channel redundant channel

**Comment by BreadcrumbIsTaken** [comment]
Done :)

**Pull request merged**